easybuild/easyblocks/m/mcr.py

```diff
@@ -147,3 +145,13 @@ def make_module_extra(self):
         txt += self.module_generator.set_environment('MCRROOT', os.path.join(self.installdir, self.subdir))
 
         return txt
+
+    def find_subdir(self):
+        # determine subdirectory (e.g. v84 (2014a, 2014b), v85 (2015a), ...)
+        import glob
+        subdirs = glob.glob(os.path.join(self.installdir,'v[0-9][0-9]*'))
+        if len(subdirs) == 1:
+            self.subdir = subdirs[0]
+        else:
+            raise EasyBuildError("Found multiple subdirectories, don't know which one to pick: %s", subdirs)
```

Comment:
Strictly speaking this error message isn't 100% accurate, since it's also possible no subdirectories are found (for example under `--module-only --force` without actually having the installation in place)

My suggestion would be to make `None` the default value for `self.subdir`, set it to `'NOTFOUND'` in case no matching subdirectories are found.

For a full installation, the sanity check should take care of this not passing through during a 'proper' installation.

(also, drop the empty line at the end)